### ISSUE TITLE
[Screen Readers-Technology Enabled Continuous Auditing-Sub Audit Template]:NVDA focus is not moving to pop up  error message upon invoking 'Submit'Button leaving the mandatory fields blank.

### ISSUE THREAD

**Issue by Amit8527510735** [microsoft/BotFramework-Composer#3505]
URL: https://auditbotui-ppe.azurewebsites.net/

Image

Actual Result:
NVDA focus is not moving to pop up  error message upon invoking 'Submit'Button leaving the mandatory fields blank.
Observation: When 'Submit' button is invoked leaving the mandatory fields blank then the NVDA focus remains on the 'Submit Button' itself.

Expected Result:
NVDA focus should move  to pop up  error message upon invoking 'Submit'Button leaving the mandatory fields blank.
Ex:When the Submit is invoked NVDA should narrate as "Information, Please correct the following errors, TRID cannot be empty, TRID should be greater than zero, Purchase Order Number cannot be empty"

Note:  For a visual user, error caused fields should be highlighted with red color border indicating they need come modification/they need attention.

Other Instances:
Same issue  is observed in the below screens also:
1. SG Membership Request& SG Membership  With File Upload
Image

When the Submit is invoked by leaving the mandatory fields blank, The NVDA focus should move to first error field and NVDA should narrate as Information, Please correct the following errors, TRID cannot be empty, TRID should be greater than zero, Please enter at least one security group names that you want to enlist.
Note:  For a visual user, error caused fields should be highlighted with red color border indicating they need come modification/they need attention.

MAS Reference:
MAS 3.3.1 - Error Identification

**Comment by cwhitten** [comment]
@Amit8527510735 where are your screenshots coming from? That is not Composer.

**Comment by Amit8527510735** [comment]
@cwhitten  We have received this bug from CSEO team, as per CSEO team this is external to BotFramework team. That's why we have logged bug under this database.
